playwright/tests/homePageElements.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { Given, Then } from "@cucumber/cucumber";
+import { expect } from "@playwright/test";
+import { CustomWorld } from "../support/setup";
+
+Given("I am on the DRB home page", async function (this: CustomWorld) {
+  return await this.page.goto("https://digital-research-books-beta.nypl.org/");
+});
+
+Then("I see the site name", async function (this: CustomWorld) {
+  await expect(
+    this.page.locator("//h1[contains(text(), 'Digital Research Books')]")
+  ).toBeVisible();
+});
+
+Then("I see the Home breadcrumb", async function (this: CustomWorld) {
+  await expect(
+    this.page.locator(
+      "//span[@class='breadcrumb-label'][contains(text(), 'Home')]"
+    )
+  ).toBeVisible();
+});
+
+Then("I see the Research breadcrumb", async function (this: CustomWorld) {
+  await expect(
+    this.page.locator(
+      "//span[@class='breadcrumb-label'][contains(text(), 'Home')]"
```

Comment:
replaced with:
//ol/li[2]/a/span[contains(text(), 'Research')]

what do you think?

Reply:
Similar to above, but using a little more specific selector because there are multiple `a` tags with the text "Research". Let's do `this.page.locator("[href='https://www.nypl.org/research']", {hasText: "Research"})`

Reply:
done.